judge/ratings.py

```diff
@@ -125,13 +145,13 @@ def rate_contest(contest):
     rating_sorted = rating_subquery.order_by('-contest__end_time')
     users = contest.users.order_by('is_disqualified', '-score', 'cumtime', 'tiebreaker') \
         .annotate(submissions=Count('submission'),
-                  last_rating=Coalesce(Subquery(rating_sorted.values('rating')[:1]), 1200),
-                  volatility=Coalesce(Subquery(rating_sorted.values('volatility')[:1]), 535),
+                  last_rating=Coalesce(Subquery(rating_sorted.values('rating')[:1]), MEAN_INIT),
```

Comment:
What I meant was to set the `1200` as a constant (i.e. `RATING_INIT = 1200`), since right now having a default of `MEAN_INIT` doesn't make sense. Also, we should probably double check that `1200` is a reasonable default rating.

Reply:
Done. 1200 is a reasonable value because in the Elo-MMR rating curve, the median is 1211 and the mean is 1279.